src/SFA.DAS.FAA.Api/Controllers/SavedSearchController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using Asp.Versioning;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.FAA.Application.Vacancies.Queries.GetSavedSearches;
+
+namespace SFA.DAS.FAA.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [ApiVersion("2.0")]
+    [ApiController]
+    [Route("/api/savedSearch/")]
+    public class SavedSearchController(IMediator mediator, ILogger<SavedSearchController> logger) : ControllerBase
```

Comment:
Worth pluralising (SavedSearchesController) this to be consistent with the VacanciesController?

Reply:
Corrected the controller name to match the others